File: SpringBootApp/src/main/java/com/example/springbootapp/data/dto/BasicProductDto.java
package com.example.springbootapp.data.dto;

import com.example.springbootapp.data.entities.Enums.Gender;
import com.example.springbootapp.data.entities.Enums.ProductCategory;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class BasicProductDto {
    private String id;
    private String name;
    private String description;
    private String brand;
    private Gender gender;
    private String picUrl;
    private String club;
    private String league;
    private ProductCategory category;
}

File: ClientAdmin/app/src/main/java/com/example/clientadmin/model/dto/BasicProductDto.kt
package com.example.clientadmin.model.dto

import com.example.clientadmin.model.enumerator.Gender
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class BasicProductDto(
    val id: String,
    val name: String,
    val description: String,
    val brand: String,
    val gender: Gender,
    val picUrl: String,
    val club: String,
    val league: String
)
